BattleTowers/src/main/java/atomicstryker/battletowers/common/DungeonTweaksCompat.java

```diff
@@ -0,0 +1,104 @@
+package atomicstryker.battletowers.common;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.util.Random;
+
+import com.evilnotch.dungeontweeks.main.world.worldgen.mobs.DungeonLocation;
+import com.evilnotch.dungeontweeks.main.world.worldgen.mobs.DungeonMobs;

```

Comment:
Pretty sure the idea of compat proxies is to avoid having to IMPORT the addons?

Reply:
ok I will remove the imports but, I don't use them